src/open_inwoner/scss/components/WarningHeader/WarningHeader.scss

```diff
@@ -0,0 +1,20 @@
+.warning-header {
+  display: flex;
+  box-sizing: border-box;
+  padding: 1rem;
```

Comment:
We have a whole bunch of nice margins and distances to chose from. Instead of 1rem, i would use "padding: var(--spacing-large);" here. 
This also makes it easier to change ALL spaces at once if ever the "huisstijl" needs to change.